application/event_processor/change_request.py

```diff
@@ -42,5 +43,6 @@ def create_payload(self) -> Dict[str, Any]:
             "message": self.message,
             "replace_opening_dates_mode": self.replace_opening_dates_mode,
             "service_id": self.service_id,
+            "odscode": self.odscode,
```

Comment:
Why is the odscode being added? we don't want to change the odscode?

Reply:
for the dentists, sometimes nhsuk sends a slightly different code . we are reformatting the odscode so that DOS can understand. It was showing in logs nhsuk odscode rather than dos code